### DISCUSSION TITLE
Deserialize vec

### DISCUSSION THREAD

**Discussion by jsoneaday** [near/borsh#116] [Q&A]
I am wanting to use Borsh to serialize and deserialize a Vec<T>. Is there documentation on how to do this anywhere?

**Comment by bowenwang1996** [comment]
Assuming you've implemented `BorshSerialize` and `BorshDeserialize` for what is inside the vector, you can call `.try_to_vec` to serialize and `try_from_slice` to deserialize. They both return a `Result`.
